packages/vuetify/src/components/VTreeview/VTreeview.ts

```diff
@@ -232,17 +239,20 @@ export default mixins(
         this.updateVnodeState(key)
       }
     },
-    calculateState (node: NodeState, state: Record<string | number, NodeState>) {
-      const counts = node.children.reduce((counts: number[], child: string | number) => {
+    calculateState (node: string | number, state: Record<string | number, NodeState>) {
+      const counts = state[node].children.reduce((counts: number[], child: string | number) => {
         counts[0] += +Boolean(state[child].isSelected)
         counts[1] += +Boolean(state[child].isIndeterminate)
         return counts
       }, [0, 0])
 
-      node.isSelected = !!node.children.length && counts[0] === node.children.length
-      node.isIndeterminate = !node.isSelected && (counts[0] > 0 || counts[1] > 0)
+      const isSelected = !!state[node].children.length && counts[0] === state[node].children.length
+      const isIndeterminate = !isSelected && (counts[0] > 0 || counts[1] > 0)
```

Comment:
group this with parenthesis

Reply:
not sure what you mean here

Reply:
```js
!!state[node].children.length && (counts[0] === state[node].children.length)
```

https://www.notion.so/vuetify/General-8ef7db0ab62a49f3a2f7cec7b67f3a44#d10855e35ea742158c874e5f08cf02ba

Reply:
how does it apply here? it's neither case 1 (`cb && cb()`) nor case 2 (`this.showWeek && day < daysInMonth`)

Reply:
Do we need to have an example for every possible scenario?

Reply:
oh, you tagged the wrong line